.github/workflows/cla.yml

```
@@ -0,0 +1,22 @@
+name: "CLA Assistant"
+on:
+  issue_comment:
+    types: [created]
+  pull_request_target:
+    types: [opened,closed,synchronize]
+
+jobs:
+  CLAssistant:
+    runs-on: ubuntu-latest
```

Just to make it more consistent with other jobs.
```suggestion
  cla-assistant:
    name: CLA Assistant
    runs-on: ubuntu-latest
```